### DISCUSSION TITLE
Questions on Quasar doc

### DISCUSSION THREAD

**Discussion by tianhuat** [quasarframework/quasar#11268] [General - Components / Directives / etc] [answered]
I have two questions related to Quasar doc here:
https://github.com/quasarframework/quasar/tree/dev/docs

1. I try to find out how to get markdown work with Quasar Doc, here are some related places:
https://github.com/quasarframework/quasar/blob/dev/docs/quasar.conf.js#L49
https://github.com/quasarframework/quasar/blob/dev/docs/build/md.js#L1
https://github.com/quasarframework/quasar/blob/dev/docs/build/md-loader.js

May I know how they work together, and where is the main Vue file that has the construct that loop all the markdowns and display them on the documentation?

2. The lines below
https://github.com/quasarframework/quasar/blob/dev/docs/quasar.conf.js#L5
https://github.com/quasarframework/quasar/blob/dev/docs/quasar.conf.js#L57
are pointing to 'quasar/dist' folder, such folder doesn't exist yet, how does it work?

**Comment by hawkeye64** [comment, marked as answer]
@tianhuat Here are some more references for you:

https://github.com/quasarframework/quasar/blob/dev/docs/src/assets/menu.js <- defines the routes and pages

https://github.com/quasarframework/quasar/blob/dev/docs/src/router/routes.js <- imports menu.js and creates the routes

https://github.com/quasarframework/quasar/blob/dev/docs/build/md-loader-utils.js#L20-L44 <- converts the transformed markdown to a Vue Component

**Reply by tianhuat** [reply]
Amazing, thank you!

**Reply by hawkeye64** [reply]
The thing to remember is this happens at build time, so you are working with a Node process.
